dev-docs/source/EventWorkspaceDev.rst

```diff
@@ -0,0 +1,93 @@
+.. _Event Workspace Dev:
+
+===========================
+Event Workspace Development
+===========================
+
+.. contents::
+  :local:
+
+The following information will be useful to you if you want to write an
+:ref:`algorithm` that is :ref:`Event Workspace <EventWorkspace>` aware.
+
+Individual Neutron Event Data (TofEvent)
+########################################
+
+The TofEvent class holds information for each neutron detection event
+data:
+
+-  PulseTime: An absolute time of the pulse that generated this neutron.
```

Comment:
Actually stored as a `Kernel::DateAndTime` object which internally stores it as you say.